src/borg/archiver.py

```diff
@@ -2272,18 +2272,34 @@ def do_break_lock(self, args, repository):
 
                 # "sh:" pattern style is the default, so the following line is not needed:
                 P sh
-                R /
-                # can be rebuild
-                - /home/*/.cache
-                # they're downloads for a reason
-                - /home/*/Downloads
-                # susan is a nice person
-                # include susans home
-                + /home/susan
-                # don't backup the other home directories
-                - /home/*
-                # don't even look in /proc
-                ! /proc\n\n''')
+                # borg will consider only the files within a recursion root
+                # defined in this way: "R path/to/recursion_root"
+                R /home
+
+                # this won't be included because nowhere in the file
+                # there isn't a line like "R /" or "R /proc"
+                + /proc
+
+                # dir won't be included because of '!'
+                + /home/*/Downloads/dir
+                ! /home/*/Downloads
```

Comment:
also true, but also a bit silly: if you forbid entering Downloads, then of course it will not enter that directory.
and as it does not enter / list that directory, it can never match the .../dir pattern.

Reply:
Yeah it is a bit silly, but isn't more clear to add those lines? In this way someone can immediately understand what it does.

Reply:
understanding how stuff works needs to be in the docs.

the examples should just show correct usage of what is learned by the docs.

if you put incorrect stuff into examples, people will just copy and paste it and wonder (even if you try to put helpful comments above - people who do not read the docs will not read these comments either).